ui/components/app/snaps/snap-ui-footer/snap-ui-footer.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FunctionComponent } from 'react';
+import { Box } from '../../../component-library';
+import { useSnapInterfaceContext } from '../../../../contexts/snaps';
+
+export const SnapUIFooter: FunctionComponent = ({ children, ...props }) => {
+  const { buttonsEnabled, requireScroll } = useSnapInterfaceContext();
+
+  // Modify each SnapUIFooterButton child to include the disabled state
+  const updatedChildren = React.Children.map(children, (child) => {
```

Comment:
I don't think we need this since we consume the scroll state from the context inside the buttons themselves?

Reply:
Yeah that's correct, this was an artifact of my desperate bug fix attemps 😅, removed this component. Pulling context in the buttons works fine.